File: src/main/scala/exoskeleton/common/ExoItems.scala
package exoskeleton.common

import cpw.mods.fml.common.registry.GameRegistry
import exoskeleton.common.core._
import exoskeleton.common.item._

object ExoItems{
  val itemCoreRecon = new ItemCore(CoreRecon);
  val itemCoreSkybound = new ItemCore(CoreSkybound);
  val itemCoreGhost = new ItemCore(CoreGhost);
  val itemCoreInferno = new ItemCore(CoreInferno);
  val itemCoreReflex = new ItemCore(CoreReflex);
  val itemCoreMedic = new ItemCore(CoreMedic);
  val itemCoreBulldozer = new ItemCore(CoreBulldozer);
  val itemCoreAssassin = new ItemCore(CoreAssassin);
  val itemCoreFrost = new ItemCore(CoreFrost);
  val itemCoreMystic = new ItemCore(CoreMystic);
  val itemCoreBerzerk = new ItemCore(CoreBerzerk);

  val itemExoArmorHelm: ItemExoHelmet = new ItemExoHelmet();
  val itemExoArmorChest: ItemExoChestplate = new ItemExoChestplate();
  val itemExoArmorLegs: ItemExoLeggings = new ItemExoLeggings();
  val itemExoArmorBoots: ItemExoBoots = new ItemExoBoots();

  def init(): Unit ={
    GameRegistry.registerItem(this.itemCoreRecon, "itemCoreRecon");
    GameRegistry.registerItem(this.itemCoreSkybound, "itemCoreSkybound");
    GameRegistry.registerItem(this.itemCoreGhost, "itemCoreGhost");
    GameRegistry.registerItem(this.itemCoreInferno, "itemCoreInferno");
    GameRegistry.registerItem(this.itemCoreReflex, "itemCoreReflex");
    GameRegistry.registerItem(this.itemCoreMedic, "itemCoreMedic");
    GameRegistry.registerItem(this.itemCoreBulldozer, "itemCoreBulldozer");
    GameRegistry.registerItem(this.itemCoreAssassin, "itemCoreAssassin");
    GameRegistry.registerItem(this.itemCoreFrost, "itemCoreFrost");
    GameRegistry.registerItem(this.itemCoreMystic, "itemCoreMystic");
    GameRegistry.registerItem(this.itemCoreBerzerk, "itemCoreBerzerk");

    GameRegistry.registerItem(this.itemExoArmorBoots, "itemExoArmorBoots");
    GameRegistry.registerItem(this.itemExoArmorChest, "itemExoArmorChest");
    GameRegistry.registerItem(this.itemExoArmorHelm, "itemExoArmorHelm");
    GameRegistry.registerItem(this.itemExoArmorLegs, "itemExoArmorLegs");
  }
}

File: src/main/scala/exoskeleton/common/EXOItems.java
package exoskeleton.common;

import cpw.mods.fml.common.registry.GameRegistry;
import exoskeleton.common.item.ItemExoBoots;
import exoskeleton.common.item.ItemExoChestplate;
import exoskeleton.common.item.ItemExoHelmet;
import exoskeleton.common.item.ItemExoLeggings;

public final class EXOItems{
    public static ItemExoHelmet itemEXOHelmet = new ItemExoHelmet();
    public static ItemExoChestplate itemEXOChestplate = new ItemExoChestplate();
    public static ItemExoLeggings itemEXOLeggings = new ItemExoLeggings();
    public static ItemExoBoots itemEXOBoots = new ItemExoBoots();

    public static void initialize(){
        GameRegistry.registerItem(itemEXOHelmet, "itemEXOHelmet");
        GameRegistry.registerItem(itemEXOChestplate, "itemEXOChestplate");
        GameRegistry.registerItem(itemEXOLeggings, "itemEXOLeggings");
        GameRegistry.registerItem(itemEXOBoots, "itemEXOBoots");
    }
}